moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectECapris.js

```diff
@@ -195,7 +196,7 @@ const ProjectSummaryProjectECapris = ({
                   </Link>
                 </>
               )) ||
-              "None"
+              ""
```

Comment:
This component is also used on the funding tab to edit the ecapris ID, (https://10419_address_ui_shortcomings_status--atd-moped-main.netlify.app/moped/projects/228?tab=funding). I think you wanted to get rid of `None` everywhere, but just wanted to point this change out just i case it wasnt expected @johnclary

Reply:
ah hah—and now I'm remembering why we originally switched from " - " to "None". the eCapris ID field on the funding tab is very hard to edit now. @tillyw — this the absolute last thing — would you mind adding the same styles to the eCapris ID field on the funding tab, but don't make it full width? iirc we use `8em` as an input width in some places?